cypress/page/form-fill.page.ts

```diff
@@ -0,0 +1,67 @@
+class PersonalFormPage{
+  private firstName: string;
+  private lastName: string;
+  private email: string;
+  private gender: string;
+  private dateOfBirth: string;
+  private mobileNumber: string;
+  private hobbies: string[];
+  private currentAddress: string;
+  private state: string;
+  private city: string;
+
+  constructor(personalInformation: personalInfo) {
+    this.firstName = personalInformation.name;
+    this.lastName = personalInformation.lastName;
+    this.email = personalInformation.email;
+    this.gender = personalInformation.gender;
+    this.mobileNumber = personalInformation.mobileNumber;
+    this.gender = personalInformation.gender;
+    this.dateOfBirth = personalInformation.dateOfBirth;
+    this.hobbies = personalInformation.hobbies;
+    this.currentAddress = personalInformation.currentAddress;
+    this.state = personalInformation.state;
+    this.city = personalInformation.city;
```

Comment:
here we should initialize the selectors/locators for each field

Reply:
Fixed